.github/workflows/render_rmarkdown.yml

```diff
@@ -0,0 +1,35 @@
+on:
+  push:
+    paths: ['**.Rmd']
```

Comment:
Should we be a bit more specific here and run it only when the push was to `main`?

Reply:
Yes, that is wise.